src/Diagnostics.ModelsAndUtils/Models/Storage/DiagEntity.cs

```diff
@@ -179,5 +179,10 @@ public List<string> AnalysisTypes
         /// Gist References stored as raw string in table.
         /// </summary>
         public string GistReferencesRaw { get; set; } = string.Empty;
+
+        /// <summary>
+        /// Is detector marked for deletion in github.
+        /// </summary>
+        public bool IsMarkedForDeletion { get; set; } = false;
```

Comment:
nit: How about renaming this to be little bit for generic like 'IsDisabled'
You can think you using this column in future as well for any other potential scenarios (like temporary disabling detectors) including permanent detection of detectors.